google-cloud-pubsub/support/doctest_helper.rb

```diff
@@ -199,6 +200,19 @@ def mock_pubsub
     end
   end
 
+  doctest.before "Google::Cloud::Pubsub::Project#publish_async" do
+    mock_pubsub do |mock_publisher, mock_subscriber, async_publisher|
```

Comment:
Should `async_publisher `  be `mock_async_publisher` for consistency? It's a mouthful, but...